configs/raw-gtp.cfg

```diff
@@ -0,0 +1,206 @@
+numGameThreads = 200
```

Comment:
Turns out we already added @include support to KataGo: https://github.com/lightvector/KataGo/pull/630

I think our implementations may have strayed a bit since then (e.g. we added support for absolute paths IIRC) but I think it works without this, which is presumably why it wasn't erroring.